src/actions/getArticlesAction.js

```diff
@@ -1,10 +1,6 @@
 import axios from "axios";
-import { BASE_URL } from "../actions/constants";
-import {
-  GET_ARTICLES_ERROR,
-  GET_ARTICLES_REQUEST,
-  GET_ARTICLES_SUCCESS
-} from "./types";
+import { BASE_URL } from "./constants";
+import { GET_ARTICLES_ERROR, GET_ARTICLES_REQUEST, GET_ARTICLES_SUCCESS } from "./types";
```

Comment:
You have exceeded the number of characters for each line please refactor

Reply:
Cool